docs/products/kafka/howto/keystore-truststore.rst

```diff
@@ -0,0 +1,52 @@
+Creating keystore and truststore for Apache Kafka
+=================================================
+
+Aiven for Apache Kafka utilises TLS (SSL) to secure the traffic between its services and client applications. This means that clients must be configured with the
+right tools to be able to communicate with the Aiven services.
+
+Keystores and truststores are password-protected files accessible to the client that interacts with the service. 
+To create these files:
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_ and select your Aiven for Apache Kafka service.
+
+2. Download the **Access Key**, **Access Certificate** and **CA Certificate** 
+
+.. image:: /images/products/kafka/ssl-certificates-download.png
+    :alt: Download the Access Key, Access Certificate and CA Certificate from the Aiven console
+
+3. Save the resulting ``service.key``, ``service.cert`` and ``ca.pem`` files in a specific folder accessible by your clients.
+
+4. In the folder where the certificates are stored, use ``openssl`` utility to create the keystore with the ``service.key`` and
+   ``service.cert`` files:
+
+::
+
+    openssl pkcs12 -export       \
+        -inkey service.key       \
+        -in service.cert         \
+        -out client.keystore.p12 \
+        -name service_key
+
+.. Note::
+    The format has to be ``PKCS12`` , which is the default since Java 9.
+
+5. You will be prompted to enter a password to protect the keystore and the key
+
+6. In the folder where the certificates are stored, use the ``keytool`` utility to create the truststore with the ``ca.pem`` file as input:
+
+::
+    
+    keytool -import  \
+        -file ca.pem \
+        -alias CA    \
+        -keystore client.truststore.jks
+
+7. You will be prompted to enter a password to protect the truststores
+
+8. You will be prompted to trust the CA certificate, reply ``yes`` to confirm
+
+The result are the keystore named ``client.keystore.p12`` and truststore named ``client.truststore.jks`` that can be used for client applications configuration.
+
+.. Tip::
+
+    The keystore and truststore creation can be automated using the :doc:`Aiven CLI </docs/tools/cli>` ``avn service user-kafka-java-creds`` as documented in the :ref:`dedicated page <avn_service_user_kafka_java_creds>`.
```

Comment:
How about using active instead of passive

You can use :doc:`Aiven CLI </docs/tools/cli>` ``avn service user-kafka-java-creds`` to create  both the keystore and the truststore. Check :ref:`dedicated page <avn_service_user_kafka_java_creds>` for more details.